### PR TITLE
SerializedCellVec improve serialize peformance

### PR DESCRIPTION
**Pull request by kitaisreal**
SerializedCellVec improve serialize peformance

### CODE REVIEW

File: ydb/core/scheme/scheme_tablecell.h [hunk 1]
```diff
@@ -120,6 +120,26 @@ struct TCell {
     const char* InlineData() const                  { Y_VERIFY_DEBUG(!IsInline_); return Ptr; }
     const char* Data() const                        { Y_VERIFY_DEBUG(!IsInline_); return Ptr; }
 #endif
+
+    void CopyData(char * dst) const {
+        if (CanInline(DataSize_)) {
```
Comment (snaury) [inline ydb/core/scheme/scheme_tablecell.h:125]: It should be `IsInline_` not `CanInline`. Then you should handle fixed size cases (including 0), and offload to memcpy everything else, using `Data()` instead of `Ptr`. I'd like code to be robust to unrelated changes.

File: ydb/core/scheme/scheme_tablecell.h [hunk 2]
```diff
@@ -120,6 +120,26 @@ struct TCell {
     const char* InlineData() const                  { Y_VERIFY_DEBUG(!IsInline_); return Ptr; }
     const char* Data() const                        { Y_VERIFY_DEBUG(!IsInline_); return Ptr; }
 #endif
+
+    void CopyData(char * dst) const {
```
Comment (snaury) [inline ydb/core/scheme/scheme_tablecell.h:124]: Name like `CopyData` lacks direction, e.g. could it maybe copy data into the cell? Maybe name it `CopyDataTo(dst)`?

**Review by snaury — CHANGES_REQUESTED** [review]

File: ydb/core/scheme/scheme_tablecell.h (hunk 1 above)
Comment (kitaisreal) [reply ydb/core/scheme/scheme_tablecell.h:125]: Updated to use `IsInline_`. Althought usage of `Data()` will produce additional branches.

File: ydb/core/scheme/scheme_tablecell.h (hunk 2 above)
Comment (kitaisreal) [reply ydb/core/scheme/scheme_tablecell.h:124]: Updated.

**Review by snaury — APPROVED** [review]
